src/Core/Services/Implementations/HandlebarsMailService.cs

```diff
@@ -565,6 +566,34 @@ private async Task RegisterHelpersAndPartialsAsync()
                 var clickTrackingText = (clickTrackingOff ? "clicktracking=off" : string.Empty);
                 writer.WriteSafeString($"<a href=\"{href}\" target=\"_blank\" {clickTrackingText}>{text}</a>");
             });
+
+            Handlebars.RegisterHelper("jsonIf", (output, options, context, arguments) =>
+            {
+                // Special case for JsonElement
+                if (arguments[0] is JsonElement jsonElement)
```

Comment:
Is it dangerous to assume all JsonElements are bools?

Reply:
In fact it is. I will add additional safety for that.